domain/network/state/subnet.go

```diff
@@ -20,10 +20,51 @@ const (
 	subnetTypeFanOverlaySegment = 1
 )
 
-// AddSubnet creates a subnet.
-func (st *State) AddSubnet(
+// UpsertSubnets updates or adds each one of the provided subnets in one
+// transaction.
+func (st *State) UpsertSubnets(ctx context.Context, subnets []network.SubnetInfo) error {
+	db, err := st.DB()
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	return db.StdTxn(ctx, func(ctx context.Context, tx *sql.Tx) error {
+		for _, subnet := range subnets {
+			err := updateSubnetSpaceIDTx(
```

Comment:
drop the Tx suffix (addSubnet) doesn't have one.

Reply:
it was a bad replace probably, fixed